react-native-in-app-payments-quickstart/app/components/GreenButton.tsx

```diff
@@ -21,12 +21,18 @@ import {
 } from 'react-native';
 import PropTypes from 'prop-types';
 
-GreenButton.propTypes = {
-  onPress: PropTypes.func.isRequired,
-  text: PropTypes.string.isRequired,
+interface GreenButton {
+  onPress: any,
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.